services/src/lib.rs

```diff
@@ -22,9 +22,9 @@
     non_camel_case_types,
     non_snake_case,
     unused_mut,
-    unused_imports,
+    //unused_imports,
     dead_code,
-    missing_docs
+    //missing_docs
```

Comment:
we need this back, otherwise I will forget about them. Just add `#[allow(unused_imports, missing_docs)]` to the header of your new file and that's it

Reply:
Sorry, my bad, it's in the other crate. Yes, that's fine